### PR TITLE
llvm17, move binaries back to /bin (reverts previous move)

### PR DESCRIPTION
**Pull request by Begasus**
Also fixes building Qt5 on 32bit: https://github.com/haikuports/haikuports/pull/10453

### CODE REVIEW

**Comment by Begasus** [comment]
Current checked with Qt5 on 64bit, 32bit check running "atm". Should need further testing with other sources on both arch's.

**Comment by korli** [comment]
LGTM

**Comment by Begasus** [comment]
Still have a few KF6 frameworks to push, once done there I'll start merging these. Thanks!

File: sys-devel/llvm/llvm17-17.0.6.recipe [hunk 1]
```diff
@@ -548,6 +548,7 @@ BUILD()
 		$cmakeDirArgs \
 		-DCMAKE_BUILD_TYPE=Release \
 		$cmakeFlags \
+		-DCMAKE_INSTALL_BINDIR=bin \
```
Comment (Begasus) [inline sys-devel/llvm/llvm17-17.0.6.recipe:551]: @davidkaroly binaries are installed in $prefix/bin on 32bit here, I used the one you did in the PR for LLVM18, this fixed 32bit issues.

**Pull request merged**